### PR TITLE
Bruce/cleanup edgectl install

### PR DESCRIPTION
**Pull request by brucehorn**
Cleaned up error messages, spacing, and alignment of messages during the edgectl install process.  Also added initial messages before each long-running process so that there is no unexplained wait in the console output.

Screenshot below:

Screen Shot 2020-04-01 at 3 22 56 PM

### CODE REVIEW

File: cmd/edgectl/aes_install.go [hunk 1]
```diff
@@ -605,79 +644,46 @@ func (i *Installer) Perform(kcontext string) error {
...
-		i.ShowWrapped(noTlsSuccess)
+		i.ShowWrapped(color.Bold.Sprintf(noTlsSuccess))
 		i.show.Println()
-		loginMsg := "Determine the IP address and port number of your Ambassador service.\n"
-		loginMsg += "(e.g., minikube service -n ambassador ambassador)\n"
-		loginMsg += fmt.Sprintf(loginViaIP, "IP_ADDRESS:PORT")
+		loginMsg := "Determine the IP address and port number of your Ambassador service, e.g.\n"
+		loginMsg += color.Bold.Sprintf("$ minikube service -n ambassador ambassador)\n\n")
+		loginMsg += fmt.Sprintf(loginViaIP)
+		loginMsg += color.Bold.Sprintf("$ edgectl login -n ambassador IP_ADDRESS:PORT")
```
Comment (ark3) [inline cmd/edgectl/aes_install.go:652]: There's a superfluous close-parenthesis in the `minikube` example.

File: cmd/edgectl/aes_install.go [hunk 2]
```diff
@@ -709,20 +715,29 @@ func (i *Installer) Perform(kcontext string) error {
...
-	i.hostname = string(content)
-	i.show.Println("-> Acquiring DNS name", color.Bold.Sprintf(i.hostname))
-
 	// Wait for DNS to propagate. This tries to avoid waiting for a ten
 	// minute error backoff if the ACME registration races ahead of the DNS
 	// name appearing for LetsEncrypt.
+	i.hostname = string(content)
+	i.show.Println("-> Acquiring DNS name", color.Bold.Sprintf(i.hostname))
+
 	if err := i.loopUntil("DNS propagation to this host", i.CheckHostnameFound, lc2); err != nil {
```
Comment (ark3) [inline cmd/edgectl/aes_install.go:741]: I don't agree with this change. The comment applies to the `loopUntil` call, but you have moved it far away.

File: cmd/edgectl/aes_install.go (hunk 1 above)
Comment (brucehorn) [reply cmd/edgectl/aes_install.go:652]: I'll add the TODO as requested and check the minikube example, thanks.

File: cmd/edgectl/aes_install.go (hunk 2 above)
Comment (brucehorn) [reply cmd/edgectl/aes_install.go:741]: I'll move it to the loopUntil block, makes sense.

**Pull request merged**